R/utils.R

```diff
@@ -23,10 +23,15 @@ setMethod("validateSex",
 setMethod("variantInfo",
           "GenotypeData",
           function(gdsobj, alleles=FALSE) {
-              data.frame(variant.id=getSnpID(gdsobj),
+              x <- data.frame(variant.id=getSnpID(gdsobj),
                          chr=getChromosome(gdsobj, char=TRUE),
                          pos=getPosition(gdsobj),
                          stringsAsFactors=FALSE)
+              if (alleles) {
```

Comment:
I just made a change in GWASTools so this should work now (allele columns were missing previously)